### PR TITLE
Improved user management and admin view 👤🔐

### PR DESCRIPTION
**Pull request by erickfabiandev**
The user was configured to have more than one role, as we have discussed today, we delegate the responsability to the backend to answer the roles that the logged user can have
The administrator view already allows adding, modifying a doctor's status and collects information from the database.
At the moment in the doctor's image field there is a text input, in the next sprint the integration for image uploading will be done.
PR was also done in the backend repo.

### CODE REVIEW

File: src/components/Admin/Doctors/DoctorForm.jsx [hunk 1]
```diff
@@ -1,102 +1,247 @@
...
+              name='password'
+              id='password'
+              required
+              value={doctor.password}
+              onChange={handleChange}
+            />
+            <h3>BASIC INFORMATION</h3>
+            <label htmlFor='file'>Image</label>
+            <input
+              // type='file'
```
Comment (ht1204) [inline src/components/Admin/Doctors/DoctorForm.jsx:145]: remove commented line.

File: src/components/Admin/Doctors/DoctorForm.jsx [hunk 2]
```diff
@@ -1,102 +1,247 @@
...
+            <label htmlFor='file'>Phone</label>
+            <input
+              type='number'
+              name='phone'
+              id='phone'
+              required
+              value={doctor.phone}
+              onChange={handleChange}
+            />
+            <h4 style={{ marginTop: '1rem' }}>Specialities</h4>
```
Comment (ht1204) [inline src/components/Admin/Doctors/DoctorForm.jsx:162]: as suggestion, try to avoid inline styles.

File: src/pages/Appointment/Appointment.jsx [hunk 3]
```diff
@@ -21,6 +23,8 @@ const Appointment = () => {
     datetime_appointment: '',
     message: ''
   })
+  const dataPatient = useLoaderData()
+  console.log(dataPatient)
```
Comment (ht1204) [inline src/pages/Appointment/Appointment.jsx:27]: remove this console.log, please.

File: src/components/Admin/Doctors/DoctorForm.jsx (hunk 1 above)
Comment (erickfabiandev) [reply src/components/Admin/Doctors/DoctorForm.jsx:145]: In this sprint we will upload the image using a text input in which we will manually place the url of the image, in the next sprint we will upload the image using the cloudinary storage, and the search for the image will be using a file input.

**Review by jlvillegas04 — APPROVED** [review]
Incredible job, Erick. Each day our projects comes more to life!

**Pull request merged**